package.json

```diff
@@ -4,11 +4,11 @@
   "description": "Typescript library for interacting with DIDs",
   "main": "lib/index.js",
   "scripts": {
-    "test": "jest --coverage",
-    "build": "./node_modules/.bin/tsc -p tsconfig.json",
+    "test": "jest",
+    "build": "tsc -p tsconfig.json",
```

Comment:
Pretty sure this may cause issues if you don't have `tsc` installed gobaly?

Reply:
When using `npm run`, the `./node_modules/.bin` folder is added to the env's path, so it's unnecessary, cf https://docs.npmjs.com/cli/run-script